Cargo.toml

```diff
@@ -13,10 +13,13 @@ exclude = [".github/"]
 readme = "README.md"
 
 [features]
-linux-native = ["dep:linux-keyutils"]
+linux-native = ["keyutils"]
 apple-native = ["dep:security-framework"]
 windows-native = ["dep:windows-sys", "dep:byteorder"]
 
+keyutils = ["dep:linux-keyutils"]
```

Comment:
It was just to align with other linux features. Since you have `{sync,async}-secret-service` (and not `linux-secret-service`), I thought `keyutils` would align nicely. Especially that we have now `{sync,async}-persistent-keyutils`.

Reply:
Ah, yes, I see your point.  But I'd rather fix that by calling the new keystore `linux-native-persistent` rather than `linux-keyutils-persistent`. The reason there's no linux- in front of secret-service is that it's not linux-only.  keyutils *is* linux-only, and built in, which is why I called it linux-native.